README.md

```diff
@@ -1,2 +1,25 @@
 # orcharhino-scripts
```

Comment:
```suggestion
![Rubocop](https://github.com/ATIX-AG/orcharhino-scripts/actions/workflows/ruby.yml/badge.svg)

# orcharhino-scripts
```

Maybe also rename the workflow to `rubocop.yaml`. If you do, adjust the file name if this suggestion.